src/audio-engine/AudioEngine.js

```diff
@@ -29,6 +30,11 @@ const AudioEngine = {
                 _this.setSwing(round.swing)
             }
             for (const userBus of Object.values(round.userBuses)) {
+                //check if lowpass and highpass are positioned second and third then move them to fourth and fifth positions
+                if (userBus.fx[1].name === 'lowpass' && userBus.fx[2].name === 'highpass') {
```

Comment:
Not blocking at all, but could you use [find](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find)? Using an index might be brittle. Ignore if you want just curious

Reply:
The index is important here, the default used to be 1 & 2 but I changed them to 4 & 5, this check is for projects that were created using the old default positioning.